students/Tejal/Lesson4/mailroom2.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python3
+import sys
+import os
+import tempfile
+import time
+
+
+
+donations = {1:{'name':'John Kestrel','amount':[653772.32, 12.17, 4]},
+2:{'name':'Jeff Kingbird','amount':[877.33]},
+3:{'name':'Paul Jacobin','amount':[663.23, 43.87, 1.32]},
+4:{'name':'Mark Tanager','amount':[1663.23, 4300.87, 10432.0]},
+5:{'name':'Anna Hummingbird','amount':[4500.0, 800.0, 2350.0]},
+6:{'name':'Calvin Fannin', 'amount':[645.0]}}
```

Comment:
you can utilize donor name as a primary key here which can make your look ups easier

for example:
```
donors = {"Paul Jacobin": [100, 200]}
```